disnake/guild_scheduled_event.py

```diff
@@ -234,7 +243,14 @@ def channel(self) -> Optional[GuildChannel]:
         guild = self.guild
         return None if guild is None else guild.get_channel(self.channel_id)
 
-    async def delete(self) -> None:
+    @property
+    def image(self) -> Optional[Asset]:
+        """Optional[:class:`Asset`]: The image associated with the guild scheduled event, if any."""
```

Comment:
```suggestion
        """Optional[:class:`Asset`]: The cover image asset of the guild scheduled event, if available."""
```
keeping it somewhat consistent with other asset properties

Reply:
changed in f8e183232b4671548a98233ae0294003a440e79e